lbrynet/daemon/Daemon.py

```diff
@@ -715,8 +715,6 @@ def _publish_stream(self, name, bid, claim_dict, file_path=None, certificate_id=
         publisher = Publisher(self.session, self.lbry_file_manager, self.session.wallet,
                               certificate_id)
         parse_lbry_uri(name)
-        if bid <= 0.0:
```

Comment:
The reason this is here is because otherwise we have to possibly make a stream before lbryum would tell us something is wrong